### PR TITLE
{bio}[GCC/8.2.0-2.31.1] Jellyfish v2.3.0

### PR DESCRIPTION
**Pull request by robqiao**
Jellyfish-2.3.0-foss-2019a

### CODE REVIEW

**Comment by smoors** [comment]
Test report by @smoors
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node358.hydra.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 2.7.5
See https://gist.github.com/2f537316c1be13162728b9a590b96c92 for a full test report.

File: easybuild/easyconfigs/j/Jellyfish/Jellyfish-2.3.0-foss-2019a.eb [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+easyblock = 'ConfigureMake'
+
+name = 'Jellyfish'
+version = '2.3.0'
+
+homepage = 'http://www.genome.umd.edu/jellyfish.html'
+description = "Jellyfish is a tool for fast, memory-efficient counting of k-mers in DNA."
+
+toolchain = {'name': 'foss', 'version': '2019a'}
```
Comment (smoors) [inline easybuild/easyconfigs/j/Jellyfish/Jellyfish-2.3.0-foss-2019a.eb]: @robqiao I think you can use the `GCC` sub-toolchain instead? I don't immediately see any usage of BLAS or MPI in the sources

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
generoso - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2660 v4 @ 2.00GHz, Python 3.6.8
See https://gist.github.com/fda236e839daa76486671a22909ebbb4 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node3103.skitty.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/a03d9c4b49a3b61de6aa3ca5881d3c65 for a full test report.

**Comment by boegel** [comment]
Going in, thanks @robqiao!

**Pull request merged**